tests/acceptance/test_dcos_package.py

```diff
@@ -16,7 +16,7 @@ def test_uninstall_package_and_wait():
 def test_install_package_with_json_options():
     install_package_and_wait('chronos', None, 'big-chronos', None, {"chronos": {"cpus": 2}})
     assert get_service_task('marathon', 'big-chronos')['resources']['cpus'] == 2
```

Comment:
@colin-msphere the get_service_task is a strange function :)   'marathon' is correct here.

the 1st param is the service, the 2nd is the task-name.   Marathon is what launched it.